### ISSUE TITLE
Publicity of parliamentary activity near elections

### ISSUE THREAD

**Issue by pmargreff** [datasciencebr/serenata-de-amor#158]
Search about deputies cases when the spent in `Publicity of parliamentary activity` happen near the elections (> 120 days) when the person take part of campaign. 

[Quoting the CEAP regulamantation](https://github.com/datasciencebr/serenata-de-amor/blob/master/CEAP.md)

> Divulgação da atividade parlamentar, exceto nos 120 dias anteriores à eleição federal, estadual ou municipal, exceto se o deputado não for candidato. 

> Disclosure of the parlamentar activity, except 120 days before elections (federal, state and municipal), except if the congressman not be a candidate. 

The initial idea is only cross the data from `Publicity of parliamentary activity` subquota, but before that, we need fetch the data for each election the congressman participated, any idea about where it could be catch?

**Comment by cuducos** [comment]
Just in case: if you haven't checked this notebok probably it will help you get started: https://github.com/datasciencebr/serenata-de-amor/blob/master/develop/2016-12-07-irio-election-expenses.ipynb

Beyond that I'm aware that data from elections are widely available: take a look in #49, maybe these sources are helpful (but honestly I'm not sure).

**Comment by pmargreff** [comment]
@cuducos Yes, I think when saw this pull request, but I guess hypotheses isn't the same, on request Irio infer the companies from elections services (btw: awesome idea) and linked it in persons. My idea is simple, only get the subquota and date and check it.

I'll look at the #49, thanks. If anyone could help with more election resource, you're welcome. 
 
Another related doubt: I don't know, but can they argue the spent isn't illegal if it isn't in the freeze days period?

**Comment by cuducos** [comment]
Yesterday I read an analysis by @vilapedro on that ; ) Once he opens the PR we can close this issue.
